### PR TITLE
Add META REFRESH tag for HTML-only redirect

### PR DESCRIPTION
**Pull request by EdgeCaseBerg**
Support browsers that have javascript off via html refresh tags. And ask users to turn on JS so the documentation will work.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <title>Project Documentation</title>
+	<meta http-equiv="refresh" content="5;URL='https://swoop-inc.github.io/scala-sugar/latest/api/'" />
```
Comment (ssimeonov) [inline index.html]: Inconsistent whitespace.

File: index.html [hunk 2]
```diff
@@ -15,6 +16,9 @@
     </script>
 </head>
 <body>
+<p>You will be redirected to the documentation below in 5 seconds. Note
```
Comment (ssimeonov) [inline index.html]: The redirect code does not wait for 5 seconds so this message is a bit misleading.

Comment (EdgeCaseBerg) [inline index.html]: The HTML Meta refresh waits for 5 second though. If you shut off java-script in your browser you'll see this behavior. I could put the `p` tag into a <noscript> tag though for clarity

File: index.html (hunk 1 above)
Comment (EdgeCaseBerg) [inline index.html]: fixed

File: index.html (hunk 2 above)
Comment (EdgeCaseBerg) [inline index.html]: added noscript tag for clarity. the p tag should only display when javascript is enabled.

**Comment by ssimeonov** [comment]
:+1:

**Pull request merged**